.github/workflows/update-mqt-core.yml

```
@@ -0,0 +1,26 @@
+name: Update MQT Core
+on:
+  schedule:
+    # run once a month on the first day of the month at 00:00 UTC
+    - cron: "0 0 1 * *"
+  workflow_dispatch:
+    inputs:
+      update-to-head:
+        description: "Update to the latest commit on the default branch"
+        type: boolean
+        required: false
+        default: false
+  pull_request:
+    paths:
+      - .github/workflows/update-mqt-core.yml
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  update-mqt-core:
+    name: ⬆️ Update MQT Core
+    uses: cda-tum/mqt-workflows/.github/workflows/reusable-mqt-core-update.yml@v1.1.5
+    with:
+      update-to-head: ${{ github.event.inputs.update-to-head != '' && fromJSON(github.event.inputs.update-to-head) || false }}
```

```suggestion
      update-to-head: ${{ github.event.inputs.update-to-head || false }}
```
Small follow-up to our offline discussion the other day: the above suggestion will also just work and is much simpler 😌

---

Great, thanks!